### DISCUSSION TITLE
Output SVG document in GraphQL query

### DISCUSSION THREAD

**Discussion by huelabs** [craftcms/cms#8910] [Ideas]
Like the [svg function](https://craftcms.com/docs/3.x/dev/functions.html#svg), it would be nice to get an SVG document inline rather an URL.

**Comment by Jones-S** [comment]
So I assume there is currently no way to output svg markup via GraphQL API right?

**Reply by brandonkelly** [reply]
You could store the SVG contents in a [Preparse field](https://plugins.craftcms.com/preparse-field) that’s added to the volume’s field layout. Set its template to:

```twig
{{ svg(element) }}
```
